sentry/api/sentry.api

```diff
@@ -661,10 +661,12 @@ public abstract interface class io/sentry/IScope {
 	public abstract fun endSession ()Lio/sentry/Session;
 	public abstract fun getAttachments ()Ljava/util/List;
 	public abstract fun getBreadcrumbs ()Ljava/util/Queue;
+	public abstract fun getClient ()Lio/sentry/ISentryClient;
 	public abstract fun getContexts ()Lio/sentry/protocol/Contexts;
 	public abstract fun getEventProcessors ()Ljava/util/List;
 	public abstract fun getExtras ()Ljava/util/Map;
 	public abstract fun getFingerprint ()Ljava/util/List;
+	public abstract fun getLastEventId ()Lio/sentry/protocol/SentryId;
```

Comment:
I was thinking that we could track last event ID per scope. By default it should probably just return the value from global scope in the future.

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3346